client/core/core.go

```diff
@@ -2875,6 +2877,46 @@ func (c *Core) tipChange(assetID uint32, nodeErr error) {
 	c.updateBalances(counts)
 }
 
+func (c *Core) ShutdownWithActiveOrders() bool {
```

Comment:
The function name is a little misleading. Maybe name this function `PromptShutdown` or similar.